Dockerfile

```diff
@@ -30,4 +30,4 @@ RUN /bin/bash -c 'cd /tmp/workspace \
  && catkin_make'
 
 
-CMD ["/bin/bash", "-c", "source /opt/ros/kinetic/setup.bash && source /tmp/workspace/devel/setup.bash && roslaunch car_demo demo.launch"]
+CMD ["/bin/bash", "-c", "source /opt/ros/kinetic/setup.bash && source /tmp/workspace/devel/setup.bash && sudo chmod 777 /home/ && roslaunch car_demo demo.launch"]
```

Comment:
chmod 777 on /home is definitely not correct. What error are you trying to work around?

Reply:
This is the error I get when I dont enable write permission in /home/
```
ERROR device /dev/input/js0 doesn't exist. Skipping
ERROR device /dev/input/js1 doesn't exist. Skipping
Executing command: 
docker run -it   --rm     --runtime=nvidia   --security-opt seccomp=unconfined   rocker_osrf/car_demo_nvidia_user 
WARNING: cannot create log directory [/home/luminosity/.ros/log/7000d20a-50f9-11e9-80d7-0242ac110002]. Please set ROS_LOG_DIR to a writable location.
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
du: cannot access '/home/luminosity/.ros/log': No such file or directory

started roslaunch server http://aa4b41023a79:35377/

SUMMARY
========

PARAMETERS
 * /joy_node0/dev: /dev/input/js0
 * /joy_node1/dev: /dev/input/js1
 * /robot_description: <?xml version="1....
 * /rosdistro: kinetic
 * /rosversion: 1.12.14
 * /use_sim_time: True

NODES
  /
    fake_localization (fake_localization/fake_localization)
    gazebo (gazebo_ros/gzserver)
    gazebo_gui (gazebo_ros/gzclient)
    joy_node0 (joy/joy_node)
    joy_node1 (joy/joy_node)
    joystick_translator (car_demo/joystick_translator)
    robot_state_publisher (robot_state_publisher/robot_state_publisher)
    rviz (rviz/rviz)
    spawn_urdf (gazebo_ros/spawn_model)
    very_inaccurate_odom (tf2_ros/static_transform_publisher)

auto-starting new master
No handlers could be found for logger "roslaunch"
[master] ERROR: unable to configure logging [unable to create directory for log file [/home/luminosity/.ros/log/7000d20a-50f9-11e9-80d7-0242ac110002].
Please check permissions.]
Roslaunch got a 'No such file or directory' error while attempting to run:

rosmaster --core -p 11311 -w 3

Please make sure that all the executables in this command exist and have
executable permission. This is often caused by a bad launch-prefix.
The traceback for the exception was written to the log file
```